datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/marketplace/README.md

```diff
@@ -17,5 +18,7 @@ All `###` headers are optional to showcase what this integration includes
 ## Setup
 Specific step-by-step configuration instructions for this integration.
 
+Screenshots are okay for this section, if they help demonstrate configuration steps.
+
 ## Support
 Information about how and where to go for support for this integration
```

Comment:
@mgarabed I'm adding the Further Reading section in this template for some consistency in our docs. It may also clue folks into use the bracketed link format.

```suggestion
Information about how and where to go for support for this integration.

## Further Reading

Additional helpful documentation, links, and articles:

- [Link text][1]
- [Link text][2]

[1]: link_url
[2]: link_url
```

Reply:
@mgarabed would a Further Reading tab show up on the tile if we added a level two heading for it?

Reply:
It would not - the section would get gobbled up.

Reply:
Ah, ok but it would feed straight into the docs @alai97 ?

Reply:
@alai97 The Further Reading seems to be something that has been added to core integrations after the fact, but not included in the template?

See [here](https://github.com/DataDog/integrations-core/blob/9b1f0b7bf13b4806444b150ac4c173226af27cac/datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/%7Bcheck_name%7D/README.md#L41) for example.

Can we address adding this as a separate PR to get coverage to all the READMEs?

Reply:
@mgarabed @anawishnoff happy to take this topic as a separate PR for all coverage, but this is in our internal doc team wiki for Further Reading sections that are rendered on the doc site—https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1169328218/Further+Reading+section.